### ISSUE TITLE
#error in step 5

### ISSUE THREAD

**Issue by raihan8025** [Lexicographical/Sign-Language-Gesture-Recognition#1]
No previous checkpoints of None exist
Exiting..

This problem is showing up in step 6. I have 4 files created in the checkpoint folder named model_file.meta, model_file.index, model_file.data-00000-of-00001 and checkpoint after running rnn_train.py. But when executing rnn_eval, why cant it find the model file?

**Comment by nourach123** [comment]
i have the same problem , do you find any solution?

**Comment by raihan8025** [comment]
yes i did. what i did was, while loading the model i added weights_only="true" like as below:
model.load('checkpoints/model_file.tflearn', weights_only="true")

this solved the problem for me

**Comment by nourach123** [comment]
i tried adding weights_only="true" but always the same problem :/
